sa/storage-authority.go

```diff
@@ -174,24 +185,43 @@ func NewSQLStorageAuthority(driver string, name string) (ssa *SQLStorageAuthorit
 		bucket: make(map[string]interface{}),
 	}
 
-	err = ssa.InitTables()
-	if err != nil {
-		return
-	}
-
+	ssa.initTables()
 	return
 }
 
-func (ssa *SQLStorageAuthority) InitTables() (err error) {
-	ssa.dbMap.AddTableWithName(core.Registration{}, "registrations").SetKeys(true, "ID").SetVersionCol("LockCol")
-	ssa.dbMap.AddTableWithName(pendingauthzModel{}, "pending_authz").SetKeys(false, "ID").SetVersionCol("LockCol")
-	ssa.dbMap.AddTableWithName(authzModel{}, "authz").SetKeys(false, "ID")
+// SetSQLDebug enables/disables GORP SQL-level Debugging
+func (ssa *SQLStorageAuthority) SetSQLDebug(state bool) {
+	ssa.dbMap.TraceOff()
+
+	if state {
+		// Enable logging
+		ssa.dbMap.TraceOn("SQL: ", &SQLLogger{blog.GetAuditLogger()})
+	}
+}
+
+// initTables constructs the table map for the ORM. If you want to also create
+// the tables, call CreateTablesIfNotExists.
+func (ssa *SQLStorageAuthority) initTables() {
+	regTable := ssa.dbMap.AddTableWithName(core.Registration{}, "registrations").SetKeys(true, "ID")
+	regTable.SetVersionCol("LockCol")
+	regTable.ColMap("Key").SetMaxSize(512).SetNotNull(true)
```

Comment:
A 4096-bit RSA key would be 4096 / 8 * 1.5 = 768 bytes after base64 encoding, plus some extra for the various JSON fields. Let's make the MaxSize a full 1k to accomodate that? Filed an issue to surface these errors in a user-friendly way: https://github.com/letsencrypt/boulder/issues/244